app/navigation/AuthorisedOfficialsNavigator.scala

```diff
@@ -77,14 +77,10 @@ class AuthorisedOfficialsNavigator @Inject() (implicit frontendAppConfig: Fronte
       userAnswers: UserAnswers =>
```

Comment:
No point for Dependency Injection in this file on line + frontendAppConfig is unused. Please remove and update to `class AuthorisedOfficialsNavigator extends BaseNavigator {` on line 31. [Here](https://github.com/hmrc/charities-registration-frontend/blob/main/test/navigation/FakeNavigators.scala#L48) will need updating also to `object FakeAuthorisedOfficialsNavigator extends AuthorisedOfficialsNavigator with FakeMainNavigator`. Thanks

Reply:
Also, remove the imports when updated. Thanks